### PR TITLE
Unify on using vespalib::duration.

### PR DESCRIPTION
**Pull request by baldersheim**
@vekterli PR

### CODE REVIEW

**Comment by baldersheim** [comment]
There should be no semantic changes here. Just a pure unification to avoid that we are confused by time units.

File: config/src/vespa/config/retriever/configretriever.cpp [hunk 1]
```diff
@@ -5,16 +5,14 @@
...
 #include <cassert>
 
-using std::chrono::milliseconds;
-
 namespace config {
 
- const milliseconds ConfigRetriever::DEFAULT_SUBSCRIBE_TIMEOUT(60000);
- const milliseconds ConfigRetriever::DEFAULT_NEXTGENERATION_TIMEOUT(60000);
+const vespalib::duration ConfigRetriever::DEFAULT_SUBSCRIBE_TIMEOUT(60s);
+const vespalib::duration ConfigRetriever::DEFAULT_NEXTGENERATION_TIMEOUT(60s);
```
Comment (vekterli) [inline config/src/vespa/config/retriever/configretriever.cpp:11]: Could consider making these `constexpr`

File: config/src/vespa/config/retriever/configretriever.h [hunk 2]
```diff
@@ -102,7 +100,7 @@ class ConfigRetriever
     std::unique_ptr<SourceSpec>                      _spec;
     bool                        _closed;
     int64_t                     _generation;
-    milliseconds                _subscribeTimeout;
+    vespalib::duration                _subscribeTimeout;
```
Comment (vekterli) [inline config/src/vespa/config/retriever/configretriever.h:103]: Consider reducing indent after type change

**Review by vekterli — APPROVED** [review]
:+1:

File: config/src/vespa/config/common/configrequest.h [hunk 3]
```diff
@@ -16,14 +16,11 @@ struct ConfigState;
...
 class ConfigRequest {
-private:
-    ConfigRequest& operator=(const ConfigRequest&);
-
 public:
     typedef std::unique_ptr<ConfigRequest> UP;
 
-    ConfigRequest() { }
-    virtual ~ConfigRequest() { }
+    ConfigRequest& operator=(const ConfigRequest&) = delete;
```
[the remark this answers is not in the document]
Comment (baldersheim) [reply config/src/vespa/config/common/configrequest.h:22]: Fixed

File: config/src/vespa/config/retriever/configretriever.cpp (hunk 1 above)
Comment (baldersheim) [reply config/src/vespa/config/retriever/configretriever.cpp:11]: Compiler does not like...

File: config/src/vespa/config/retriever/configretriever.h (hunk 2 above)
Comment (baldersheim) [reply config/src/vespa/config/retriever/configretriever.h]: Fixed

**Pull request merged**